src/platform_impl/macos/window.rs

```diff
@@ -269,6 +269,7 @@ lazy_static! {
 #[derive(Default)]
 pub struct SharedState {
     pub resizable: bool,
+    pub visible: bool,
```

Comment:
this gets default initialized to `false` it seems

Reply:
Yes you are right! After fixing this, it works for me! I just jumped to assuming that it was my system at fault after the fact that the other PR didn't work either.

Reply:
I just pushed the fix; it seemed easiest to do so.